app/src/main/res/layout/activity_meal.xml

```diff
@@ -70,8 +70,58 @@
                 android:layout_marginTop="@dimen/meal_page_name_margin_top"
                 android:layout_marginBottom="@dimen/meal_page_name_margin_bottom"/>
 
+            <RelativeLayout
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/meal_page_text_margin">
```

Comment:
Can we make this a LinearLayout instead? It has slightly better performance and we don't need anything fancy here.